src/TelemetryChannels/ServerTelemetryChannel/NuGet/TelemetryChannel.NuGet.csproj

```diff
@@ -38,7 +38,7 @@
     </None>
   </ItemGroup>
   <ItemGroup>
-    <ProjectReference Include="..\..\..\Core\Managed\Net45\Core.Net45.csproj">
+    <ProjectReference Include="..\..\..\Core\Managed\Net40\Core.Net40.csproj">
```

Comment:
That is explain by Sergey in description: Description: I use the same version of `Microsoft.ApplicationInsights.dll` in both nuspec. this solves the problem of uneven versions for nugget packages

---
In reply to: [80967630](https://github.com/Microsoft/ApplicationInsights-dotnet/pull/317#discussion_r80967630) [](ancestors = 80967630)